### ISSUE TITLE
Training time

### ISSUE THREAD

**Issue by clausmichele** [swang0926/flownet_tensorflow#2]
Hi,
I am trying to use your network with just 400 couple of images from the FlyingChair dataset but it seems to take 25 days to train. How long did it take for you? With a training set of?

Regards,
Michele

**Comment by clausmichele** [comment]
Training with 8000 training samples, after 15 epochs the loss is still higher than 1000 and does not seem to decrease, any suggestion?
